src/ctim/schemas/vocabularies.cljc

```diff
@@ -265,7 +266,8 @@
     "email_messageid"
     "email_subject"
     "cisco_mid"
-    "mutex"})
+    "mutex"
+    "registry_key"})
```

Comment:
The `registry_key` and `registry_value` observable types have been removed in this PR https://github.com/threatgrid/ctim/pull/230. Did you discuss about it with @saintx?

Reply:
Yes, we talked and since the PR that was to re-introduce the registry types is on hold because of the inability to support composite observables, we can add this back for the Orbital CTR integration

Reply:
The structured registry key path, data type, and value that I describe in https://github.com/threatgrid/hydrant/wiki/Encoding-Registry-Key-Values will have to wait.  We can call it something else.  We need some kind of `registry_key` observable type for orbital, so we should add it.